tests/functional/container/test_container.py

```diff
@@ -319,6 +321,46 @@ def test_create_with_unknown_storage_policy(self):
         data = self.json_loads(resp.data)
         self.assertEqual(data["status"], 480)
 
+    def _test_create_with_status(self, status=None):
+        def _status(_data):
+            return data['system']['sys.status']
```

Comment:
I think you mean `_data['system']['sys.status']` (with the undescore). This unexpectedly works because of Python's variable scope.